<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEvent</class>
 <widget class="QDialog" name="DialogEvent">
  <property name="windowTitle" stdset="0">
   <string>Dialog</string>
  </property>
  <property name="geometry" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>434</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelSummary">
       <property name="text" stdset="0">
        <string>Summary:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSummary"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelLocation">
       <property name="text" stdset="0">
        <string>Location:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLocation"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelDescription">
       <property name="text" stdset="0">
        <string>Description: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditDescription"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelStarts">
       <property name="text" stdset="0">
        <string>Starts: </string>
       </property>
       <property name="alignment" stdset="0">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEditStart"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelStartHour">
       <property name="text" stdset="0">
        <string>Hour (24h):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBoxStartTimeHour">
       <property name="value" stdset="0">
        <number>0</number>
       </property>
       <property name="minimum" stdset="0">
        <number>0</number>
       </property>
       <property name="maximum" stdset="0">
        <number>23</number>
       </property>
       <property name="toolTip" stdset="0">
        <string>Start Hour</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="labelStartMin">
       <property name="text" stdset="0">
        <string>Mins:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSpinBox" name="spinBoxStartTimeMinute">
       <property name="maximum" stdset="0">
        <number>59</number>
       </property>
       <property name="toolTip" stdset="0">
        <string>Start Minute</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelEnds">
       <property name="text" stdset="0">
        <string>Ends: </string>
       </property>
       <property name="alignment" stdset="0">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QDateEdit" name="dateEditEnd"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelEndHour">
       <property name="text" stdset="0">
        <string>Hour (24h):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QSpinBox" name="spinBoxEndTimeHour">
       <property name="maximum" stdset="0">
        <number>23</number>
       </property>
       <property name="toolTip" stdset="0">
        <string>End Hour</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="labelEndMin">
       <property name="text" stdset="0">
        <string>Mins:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="2">
      <widget class="QSpinBox" name="spinBoxEndTimeMinute">
       <property name="maximum" stdset="0">
        <number>59</number>
       </property>
       <property name="toolTip" stdset="0">
        <string>End Minute</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons" stdset="0">
        <set>QDialogButtonBox::NoButton</set>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxAllDay">
       <property name="text" stdset="0">
        <string>All Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRepeatsYearly">
       <property name="text" stdset="0">
        <string>Repeats Yearly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxPriority">
       <property name="text" stdset="0">
        <string>High Priority</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="checkBoxReminder">
       <property name="text" stdset="0">
        <string>Set Reminder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxReminder"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="checkBoxDelete">
       <property name="text" stdset="0">
        <string>Delete Event</string>
       </property>
       <property name="font" stdset="0">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation" stdset="0">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="pushButtonOk">
         <property name="text" stdset="0">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text" stdset="0">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEvent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEvent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
